### DISCUSSION TITLE
Set up different user email for a specific project

### DISCUSSION THREAD

**Discussion by tiguchi** [theory/sqitch#790] [Q&A] [answered]
I ran into a situation where I would like to configure a specific Sqitch project with a different user email address (other than the one already defined in my home directory's `~/.sqitch/sqitch.conf` file). I cannot add the email address to the project's `sqitch.conf` file though, since it's in source control and shared with a team.

In Git it's possible to specify different configurations per path as follows:

File `~/.gitconfig`
```
[includeIf "gitdir:~/path/to/project"]
    path = .gitconfig-alternative
```

Is there a similar way to configure sqitch?

**Comment by theory** [comment, marked as answer]
There is not. Might be nice to add `--planner-name` and `--planner-email` options to the `add` and `rework` commands. In the meantime you can set the `SQITCH_FULLNAME` and `SQITCH_EMAIL` environment variables when you run `sqitch add` or `sqitch rework`. Or you can edit the plan file immediate after you add a change (but before you deploy it).

Check out [`sqitch help environment`](http://sqitch.org/docs/manual/sqitch-environment/) for more goodies.
